src/olympia/reviewers/models.py

```diff
@@ -383,21 +383,32 @@ def base_query(self):
 class ReviewerSubscription(ModelBase):
     user = models.ForeignKey(UserProfile, on_delete=models.CASCADE)
     addon = models.ForeignKey(Addon, on_delete=models.CASCADE)
+    channel = models.IntegerField(choices=amo.RELEASE_CHANNEL_CHOICES)
 
     class Meta:
         db_table = 'editor_subscriptions'
 
     def send_notification(self, version):
         user_log.info('Sending addon update notice to %s for %s' %
                       (self.user.email, self.addon.pk))
+
+        if version.channel == amo.RELEASE_CHANNEL_LISTED:
+            listing_url = absolutify(reverse('addons.detail',
+                                             args=[self.addon.pk],
+                                             add_prefix=False))
+        else:
+            # If the submission went to the unlisted channel,
+            # do not link to the listing.
+            listing_url = '<unlisted submission>'
```

Comment:
```suggestion
            listing_url = None
```

Reply:
I wanted to be explicit so one doesn't assume it's a bug because it's missing, but okay.